Disable lower TargetFrameworkVersion forcing to avoid compatibility problems
between gtk-sharp compiled with latest Cairo and older Cairo for mono < 4.5.
--- mono-addins-mono-addins-1.0/Mono.Addins.Gui/Mono.Addins.Gui.csproj.orig	2014-01-07 21:11:14.359992561 +0100
+++ mono-addins-mono-addins-1.0/Mono.Addins.Gui/Mono.Addins.Gui.csproj	2014-01-07 21:11:23.459992181 +0100
@@ -11,7 +11,6 @@
     <RootNamespace>Mono.Addins.Gui</RootNamespace>
     <SignAssembly>True</SignAssembly>
     <AssemblyOriginatorKeyFile>..\mono-addins.snk</AssemblyOriginatorKeyFile>
-    <TargetFrameworkVersion>v3.5</TargetFrameworkVersion>
   </PropertyGroup>
   <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Debug|AnyCPU' ">
     <DebugSymbols>True</DebugSymbols>
